include/WebServer.hpp

```diff
@@ -28,6 +28,7 @@
 #define LOCAL_DIR			"directory"
 #define INDEX				"index"
 #define ERRORPAGE_LOCATION	"error_page"
+#define DEFAULT_NAME		"default"
```

Comment:
smallest thing ever but maybe something lie kdefault_server_name would be more descriptive? no need to change now though, splitting hairs

Reply:
done